portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,126 @@
+/*---Images---*/
+.profilePic {
+    border-radius: 50%;
+    border: 8px solid #28ACEA;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+.socialMediaIcons {
+    width: 45px;
+    height: 45px;
+    margin: 0px 5px 0px 5px;
+}
+.experienceLogos {
+    border-radius: 15%;
+    width: 180px;
+    height: 180px;
+    margin-right: 20px;
+}
+
+/*---Fonts---*/
+.fonts {
+    font-family: "Gill Sans";
+    font-weight: 150;
+    font-size: 20px;
+}
+.mediumFont {
+    font-family: "Gill Sans";
+    font-weight: 150;
+    font-size: 25px;
+    padding-bottom: 0px;
+}
+.headerFont {
+    font-family: "Gill Sans";
+    font-weight: 500;
+    font-size: 30px;
+}
+
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  text-align: center;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+
+/* #body {
+    background-image: url('/images/cottonCandyBackground.jpeg');
+    background-size: cover;
+    background-position: center;
+    position: relative;
+} */
+
+
+.center {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+
+
+.test {
```

Comment:
Rename to something that describes what this is, or should be used for. 

For example, it looks like this is used to help format div containers for each header. Maybe something like `.content-div`